lib/resources/trending_movies_api_provider.dart

```diff
@@ -19,7 +19,28 @@ class TrendingMoviesApiProvider {
         json.decode(response.body),
       );
     } else {
-      throw Exception("Failed to get data");
+      throw Exception(StringConstants.exceptionText);
+    }
+  }
+
+  Future<MovieModel> searchMovies(String query) async {
+    final response = await http.get(
+      Uri.parse(
+        StringConstants.uri + StringConstants.searchMoviesEndpoint + query,
+      ),
+      headers: {
+        "Content-Type": StringConstants.contentType,
+        "Authorization": StringConstants.apiToken,
+      },
+    );
+    if (response.statusCode == 200) {
```

Comment:
the 200 can be added to the constants file

Reply:
Added.